frontend/packages/dev-console/src/components/pipelineruns/PipelineRunsResourceList.tsx

```diff
@@ -6,18 +6,23 @@ import { PipelineRunModel } from '../../models';
 import { runFilters } from '../pipelines/detail-page-tabs/PipelineRuns';
 import PipelineRunsList from './list-page/PipelineRunList';
 
+interface PipelineRunsResourceListProps {
+  showBadge?: boolean;
```

Comment:
`hideBadge`? That way you don't need to worry about a default. `showBadge` also implies it has some baring over a badge, whereas it's more or less just hiding a badge that _may_ want to show. Semantics, not really important - consider it a nit.